apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue

```diff
@@ -210,15 +212,14 @@ const handleDynamicLayoutFetch = (changed) => {
 /** ******* Page Init ******* */
 const getTableSchema = async () => {
     try {
-        const schema = await SpaceConnector.client.addOns.pageSchema.get({
+        tableState.schema = await SpaceConnector.client.addOns.pageSchema.get({
             // eslint-disable-next-line camelcase
             resource_type: 'identity.ServiceAccount',
             schema: 'table',
             options: {
                 provider: state.selectedProvider,
             },
-        });
-        tableState.schema = schema;
+        }, { mockPath: '?resource=serviceaccount' });
```

Comment:
nitpick: It might be safer to use a TODO comment here!

Reply:
Oh... Thank you